internal/driver/deviceclient.go

```diff
@@ -64,28 +64,28 @@ func createConnectionInfo(addr models.Addressable) (*ConnectionInfo, error) {
 }
 
 func createCommandInfo(object models.DeviceObject) *CommandInfo {
-	var primaryTable, _ = object.Attributes["primaryTable"].(string)
-
-	var startingAddress, _ = strconv.Atoi(object.Attributes["startingAddress"].(string))
-	var startingAddressUint16 = uint16(startingAddress - 1)
+	primaryTable, _ := toString(object.Attributes["primaryTable"])
+	startingAddress, _ := toUint16(object.Attributes["startingAddress"])
```

Comment:
why those two error variables are ignored?  They should be handled if something goes wrong

Reply:
I check these attributes at https://github.com/edgexfoundry/device-modbus-go/pull/10/files#diff-54a06bb7cfa1776ea4ef16cfe03e9cedR35.